compiler/src/iree/compiler/Codegen/Utils/Utils.h

```diff
@@ -252,6 +252,34 @@ bool isFullSlice(OffsetSizeAndStrideOpInterface sliceLoadStoreOp,
                  IREE::Flow::DispatchTensorType tensorType,
                  ValueRange dynamicDims);
 
+//===----------------------------------------------------------------------===//
+// Utility functions for vector size inference for dynamic shapes
+//===----------------------------------------------------------------------===//
+
+struct VectorizationTileSizes {
+  SmallVector<int64_t> destShape;
+  SmallVector<int64_t> vectorSizes;
+  SmallVector<bool> vectorScalableFlags;
+};
+
+/// Returns a VectorizationTileSizes which contains the inferred bounded result
+/// shape and vector input sizes. This is useful to infer the sizes from a
+/// chain.
+std::optional<VectorizationTileSizes> inferSizesFromIR(Value val);
+/// Returns the result sizes and vector input sizes of the tensor.unpack op. The
```

Comment:
Final nits: add a blank line before the comment. It usually helps readability. IREE also follows this style in most of headers.

https://google.github.io/styleguide/cppguide.html#Vertical_Whitespace

Reply:
done :)